modules/core/src/test/java/org/apache/ignite/cache/affinity/AffinityHistoryCleanupTest.java

```diff
@@ -89,34 +89,34 @@ public void testAffinityHistoryCleanup() throws Exception {
 
             Ignite ignite = startGrid(0);
 
-            checkHistory(ignite, F.asList(topVer(1, 0)), 1);
+            checkHistory(ignite, F.asList(topVer(1, 0)), 1); //fs = 1
```

Comment:
What does "fs" mean? Please use more detailed comments.

Reply:
I mean FullHistorySize. I will rename it to something more clear.